Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:18.04
+
+#Install all the requirements
+RUN apt-get update && apt-get install -y curl git build-essential software-properties-common wget zlib1g-dev libssl1.0-dev libreadline-dev libyaml-dev libffi-dev libxml2-dev libxslt1-dev phantomjs wait-for-it
```

Comment:
I think we don't use `phantomjs` anymore.

Reply:
I will try to remove it then, was getting it from the wiki.